omnibus/config/software/librdkafka.rb

```diff
@@ -21,6 +22,7 @@
   env = with_standard_compiler_flags(with_embedded_path)
   configure_options = [
     "--enable-sasl",
+    "--enable-curl=y"
```

Comment:
Just curious, why `=y`?
```suggestion
    "--enable-curl"
```

Reply:
The intent is to have the build fail if libcurl isn't available. Otherwise it default to "auto" which usually means "enabled if the dependency is found, disabled if it isn't found"